Assets/MixedRealityToolkit.SDK/Features/UX/Meshes/Cursors/Materials/Light.mat

```diff
@@ -9,8 +9,7 @@ Material:
   m_PrefabAsset: {fileID: 0}
   m_Name: Light
   m_Shader: {fileID: 4800000, guid: 5bdea20278144b11916d77503ba1467a, type: 3}
-  m_ShaderKeywords: _DIRECTIONAL_LIGHT _DISABLE_ALBEDO_MAP _HOVER_LIGHT _REFLECTIONS
```

Comment:
I don't know where these changes came from, must have accidentally committed. I'll revert.

Reply:
Oh, these changes came from @cre8ivepark's fork. Yoon, do we need these changes and the changes to Shadow.mat?

Reply:
Yes, updated both Light.mat and Shadow.mat to make the cursor always visible by ignoring the lighting